charts/brigade/values.yaml

```diff
@@ -161,6 +167,12 @@ service:
   type: LoadBalancer
   externalPort: 7744
   internalPort: 7744
+  # Configure liveness probes except `httpGet`
```

Comment:
It looks like this section should either be moved up under the `gw` section (as current commit has the `gateway-github-deployment.yaml` referencing `.Values.gw.livenessProbe`, etc.) or the aforementioned references updated to use `.Values.service.livenessProbe`, etc.